### PR TITLE
Bah 19871

### PR DESCRIPTION
**Pull request by elder-joseph**
## Description
As a comparison tool user, I need the the contact details information to be displayed in a clear and concise manner so that I can easily identify the information I need when contacting a provider or SCO.

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/19871

## Testing done
Manual and unit testing passes locally

## Screenshots
N/A

## Acceptance criteria Specific to this PR
- [x] The email address of the provider is displayed.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: app/serializers/institution_program_profile_serializer.rb [hunk 1]
```diff
@@ -11,5 +11,6 @@ class InstitutionProgramProfileSerializer < ActiveModel::Serializer
              :school_locale,
              :tuition_amount,
              :va_bah,
-             :dod_bah
+             :dod_bah,
+             :provider_email_address
```
Comment (shawkey-daniel) [inline app/serializers/institution_program_profile_serializer.rb]: Small nit, can you move provider_email_address next to provider_website?

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

File: app/serializers/institution_program_profile_serializer.rb (hunk 1 above)
Comment (elder-joseph) [reply app/serializers/institution_program_profile_serializer.rb]: No problem, pushed and updated commit with the re-ordering

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

**Review by jholton — APPROVED** [review]
LGTM

**Pull request merged**